components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -4,13 +4,16 @@
 
 package mozilla.components.browser.engine.gecko
 
+import kotlinx.coroutines.experimental.CompletableDeferred
+import kotlinx.coroutines.experimental.runBlocking
 import mozilla.components.concept.engine.EngineSession
 import org.mozilla.geckoview.GeckoResponse
 import org.mozilla.geckoview.GeckoRuntime
 import org.mozilla.geckoview.GeckoSession
 
 const val PROGRESS_START = 25
 const val PROGRESS_STOP = 100
+const val GECKO_STATE_KEY = "GECKO_STATE"
```

Comment:
This one (and probably the others) may not need to be public global constants and could just sit in the companion object of GeckoEngineSession, I guess?

Reply:
Good catch. I'll update.